src/datapoint.jl

```diff
@@ -66,16 +71,21 @@ const ±(val, err::NamedTuple) = DataPoint(val, hasproperty(err,:stat) ? err.sta
 const ±(D::DataPoint, sys) = DataPoint(D.val, D.stat_err, sys)
 
 ## Convert DataPoint-types
-convert(::Type{DataPoint{T}}, D::DataPoint) where {T} = DataPoint{T}(T( D.val),T.(D.stat_err), T.(D.sys_err))
+convert(::Type{DataPoint{T}}, D::DataPoint) where {T} = DataPoint{T}(T(D.val),T.(D.stat_err), T.(D.sys_err))
 convert(::Type{DataPoint{T}}, D::DataPoint) where {T<:Integer} = DataPoint{T}(T(D.val),T.(ceil.(D.stat_err)), T.(ceil.(D.sys_err)))
 
+## DataPoint math
+Base.:/(D::DataPoint, s::T) where {T <: Number} = DataPoint(D.val/s,D.stat_err/s,D.sys_err/s)
+Base.:*(D::DataPoint, s::T) where {T <: Number} = DataPoint(D.val*s,D.stat_err*s,D.sys_err*s)
+
 ## Pretty-printing
 # Printing of results
 function show(io::IO, ::MIME"text/plain", D::DataPoint{T}; drop::Bool = true) where {T}
     println(io, "DataPoint{",T,"}:")
-    print(io, "Value → ", D.val)
-    if (!drop || D.stat_err != (0,0)) print(io, "\tStatistic unc. → ", (1,-1) .* D.stat_err); end
-    if (!drop || D.sys_err != (0,0)) print(io, "\tSystematic unc. → ", (1,-1) .* D.sys_err); end
+    print(io, "Value → ")
+    show(io, D.val)
+    if (!drop || D.stat_err != (0,0)) print(io, "\tStatistic unc. → ", (-1,1) .* D.stat_err); end
```

Comment:
Maybe printing them as `val +- stat +- sys` is more space efficient and easier to read? But that can go in a separate PR.

Reply:
Yes indeed I am not happy with the printing at this moment, but this is something for another PR.
Actually when the `io::IO` is `:compact` (e.g. in arrays) then it already prints your proposed way.